src/providers/postgres/raster/qgspostgresrastershareddata.cpp

```diff
@@ -304,7 +304,7 @@ bool QgsPostgresRasterSharedData::fetchTilesIndex( const QgsGeometry &requestPol
   // Include actual bounds
   if ( !overallExtent.isNull() )
   {
-    mLoadedIndexBounds[ cacheKey ] = requestPolygon.combine( QgsGeometry::fromRect( overallExtent ) );
+    mLoadedIndexBounds[ cacheKey ] = requestPolygon.combine( QgsGeometry::fromWkt( overallExtent.asWktPolygon() ) );
```

Comment:
Why use `fromWkt`? You can “lose” precision during the conversion double->str->double

Reply:
Thanks. This was a regression from a bad conflict resolution while rebasing the PR.